framework/src/Volo.Abp.Quartz/Volo/Abp/Quartz/AbpQuartzModule.cs

```diff
@@ -20,9 +21,15 @@ public override void ConfigureServices(ServiceConfigurationContext context)
 
         public override void OnApplicationInitialization(ApplicationInitializationContext context)
         {
+            var options = context.ServiceProvider.GetRequiredService<IOptions<AbpQuartzPreOptions>>().Value;
+            
             _scheduler = context.ServiceProvider.GetService<IScheduler>();
             _scheduler.JobFactory = context.ServiceProvider.GetService<IJobFactory>();
-            _scheduler.Start();
+
+            if (options.StartDelay != null && options.StartDelay.Ticks > 0)
+                _scheduler.StartDelayed(options.StartDelay);
+            else
+                _scheduler.Start();
```

Comment:
Please add curly brackets to keep the code style.

`StartDelay `is never be `null`.